### PR TITLE
Add Movary

### PR DESCRIPTION
**Pull request by DrMxrcy**
(No Image so I used the Tipi Logo as a placeholder)

### CODE REVIEW

**Comment by meienberger** [comment]
Can we also find a logo for this one?

**Comment by DrMxrcy** [comment]
> Can we also find a logo for this one?

I did something similar to Tasks.md, found a close emoji and resized it.

File: apps/movary/metadata/logo.jpg
Comment (meienberger) [inline apps/movary/metadata/logo.jpg:1]: Can you also unzoom this icon?

**Review by meienberger — CHANGES_REQUESTED** [review]

File: apps/movary/metadata/logo.jpg
Comment (DrMxrcy) [reply apps/movary/metadata/logo.jpg:1]: Fixed in da9f708, Used a background as well.

File: apps/movary/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    labels:
+      traefik.enable: ${APP_EXPOSED}
+      traefik.http.routers.movary.rule: Host(`${APP_DOMAIN}`)
+      traefik.http.routers.movary.entrypoints: websecure
+      traefik.http.routers.movary.service: movary
+      traefik.http.routers.movary.tls.certresolver: myresolver
+      traefik.http.services.movary.loadbalancer.server.port: 80
+
+  movary-mysql:
+    image: mysql:8.0
```
[the remark this answers is not in the document]
Comment (DrMxrcy) [reply apps/movary/docker-compose.yml]: Fixed in da9f708

**Review by meienberger — CHANGES_REQUESTED** [review]

**Review by meienberger — APPROVED** [review]

**Pull request merged**